pkg/ebpf/tracee.go

```diff
@@ -162,10 +162,16 @@ type fileExecInfo struct {
 	Hash      string
 }
 
+type handledEvent struct {
```

Comment:
I think the name doesn't fit very well, `handledEvent` to me implies the struct is some transformation of the event, but this struct describes a strategy of handling events if I understand correctly.
Maybe `eventHandleConfig`?